lib/ClimaCoreTempestRemap/src/onlineremap.jl

```diff
@@ -2,7 +2,7 @@ import ClimaCore.DataLayouts: IJFH
 
 
 """
-    LinearTempestRemap{T, S, M, C, V}
+    ClimaCoreTempestRemap.LinearMap{T, S, M, C, V}
```

Comment:
I don't think we need to qualify these methods in the doc string, Documenter already provides a qualified name above the doc string (see the API docs for an example).

The upside is that the name exactly matches, the downside is that all the function doc strings get a bit too wordy. Also, it's not a pattern that we're currently using across this and other repos (it doesn't seem to me to be a common pattern in other Julia packages either)

Reply:
This was one of the "missing docrstring" errors that I tried to chase down and saw that this was a frequent issue among other Documenter users and found that qualifying would solve it. It did solve it in this instance

Reply:
We can do qualifying in the docs API

Reply:
Yeah, but then again, it was my attempt, before discovering that actually the docstring had an outdated name! Before, apparently it used to be called `LinearTempestRemap` and then it was later changed to `LinearMap` and people forgot to update the docstring

Reply:
So it's probably ok now to remove it altogether. I am going to try

Reply:
We should probably update https://github.com/CliMA/ClimaCore.jl/blob/main/docs/make.jl#L59 to [:example_block, :export] (if that's the syntax), to ensure this doesn't keep popping up.

Reply:
@charleskawczynski , According to [Documenter's docs](https://documenter.juliadocs.org/stable/lib/public/index.html#Documenter.makedocs), the keywords are: `:autodocs_block`, `:cross_references`, `:docs_block`, `:doctest`, `:eval_block`, `:example_block`, `:footnote`, `:linkcheck`, `:meta_block`, `:missing_docs`, `:parse_error`, and `:setup_block`. So there is no `:export` or similar. What did you have in mind?